gradle-plugin/README.md

```diff
@@ -95,8 +95,8 @@ appservices {
         "reference-browser" {
             moduleIdentifier 'org.mozilla.appservices:reference-browser-megazord'
             components.clear()
-            component 'org.mozilla.fxaclient:fxaclient'
-            component 'org.mozilla.sync15:logins'
+            component 'org.mozilla.appservices', 'fxaclient'
```

Comment:
Wait, is this right?

Reply:
https://github.com/mozilla/application-services/blob/master/gradle-plugin/src/main/kotlin/mozilla/appservices/MegazordDefinition.kt#L44-L50 yup!